### PR TITLE
hotfix(sentry): PKCS#8 encode private key

### PR DESCRIPTION
**Pull request by 1046102779**

### CODE REVIEW

**Comment by yaron2** [comment]
What does this fix?

**Comment by 1046102779** [comment]
PKCS#8 encode EC Private Key

File: pkg/sentry/csr/csr.go [hunk 1]
```diff
@@ -187,7 +187,7 @@ func encode(csr bool, csrOrCert []byte, privKey *ecdsa.PrivateKey, pkcs8 bool) (
 	var err error
 
 	if pkcs8 {
-		if encodedKey, err = x509.MarshalECPrivateKey(privKey); err != nil {
+		if encodedKey, err = x509.MarshalPKCS8PrivateKey(privKey); err != nil {
 			return nil, nil, err
 		}
 		privPem = pem.EncodeToMemory(&pem.Block{Type: blockTypePrivateKey, Bytes: encodedKey})
```
Comment (artursouza) [inline pkg/sentry/csr/csr.go:193]: How was this working before? Is this path not exercised? Unit test here will help.

Comment (1046102779) [reply pkg/sentry/csr/csr.go:193]: ok. i write a UT

**Comment by artursouza** [comment]
@1046102779 Any updates on this?

**Pull request closed without merging**